### ISSUE TITLE
Unable to harvest or Synchronise metadata with GISC Tokyo

### ISSUE THREAD

**Issue by woollattd** [OpenWIS/openwis#292]
Since the https update to the Portal/oaipmh URL for GISC Tokyo (1st March 2018), OpenWIS at GISC Exeter has been unable to connect to them for the Synchronisation or Harvesting of metadata.
This issue has also been confirmed to be happening at GISC Toulouse and GISC Washington by Benjamin and Steve.
Is this an issue with OpenWIS or with GISC Tokyo?

The fetch function on the harvest or synchronise screen doesn't return any results.

**Comment by woollattd** [comment]
it seems that GISC Casablanca was having some issues harvesting GISCs Tokyo through OAIPMH.
Their Harvesting now works with GISC Tokyo after updating their scripts to manage https communications

is there something we need to do in OpenWIS?

**Comment by mgiannoni** [comment]
Dom:

Yes, we encountered this when UK-Met went HTTPs.   The harvester does not
support HTTPs, I believe there should be an issue already opened for this -
but I will have to search for that.  What we did was implement a simple
python HTTPs proxy script and set the OAI-MPH harvester (for UK-Met) to
connect to the python proxy.


Marc Giannoni
Unix System Engineer, AceInfo Solutions
Phone: 301.427.9478
Cell: 301.915.5266
